build/uploadperftobenchview.cmd

```diff
@@ -72,6 +72,7 @@ py "%perfWorkingDirectory%\Microsoft.BenchView.JSONFormat\tools\submission.py" "
                     --type "%runType%" ^
                     --config-name "%configuration%" ^
                     --config Configuration "%configuration%" ^
+                    --config OS "%OS%" ^
```

Comment:
@DrewScoggins @jorive Does the submission script support specifying multiple `--config` parameters?

Reply:
Yes, it supports specifying multiple `--config <key, value>` arguments.